framework/include/samplers/Sampler.h

```diff
@@ -110,6 +112,8 @@ class Sampler : public MooseObject,
   dof_id_type getLocalRowBegin() const;
   dof_id_type getLocalRowEnd() const;
   ///@}
+  //
+  void initForMultiApp(const MultiApp * multiapp);
```

Comment:
This could use some explanation.
```suggestion
  void initForMultiApp(const MultiApp * multiapp);
```

Reply:
added